alfalfa_worker/lib/data/data_generator.py

```diff
@@ -164,17 +162,15 @@ def generate_weather(self,
         str_old = 'filNam=""'
         str_new = 'filNam=Modelica.Utilities.Files.loadResource("{0}")'\
                   .format(self.weather_file_name)
-
         with open(model_file) as f:
             newText = f.read().replace(str_old, str_new)
 
         with open(model_file, "w") as f:
             f.write(newText)
 
         # Change to Resources directory
-        currdir = os.getcwd()
+        currdir = os.curdir
         os.chdir(self.weather_dir)
```

Comment:
Is there a way we can do this stuff without changing dir?

Reply:
I'm sure there is, but I really don't want to touch this code until we can get adequate testing implemented on this file. I actually plan on removing `data_generator` for our [`fmu_runner.py`](https://github.com/urbanopt/geojson-modelica-translator/blob/develop/geojson_modelica_translator/modelica/lib/runner/fmu_runner.py), eventually.

Reply:
Makes sense. I currently have the dispatcher reset the CWD every time it gets a new job (due to MLEP changing directory) so this should be fine